Weather App/controller/SearchTabViewController.swift

```diff
@@ -0,0 +1,89 @@
+import SnapKit
+import UIKit
+
+class SearchTabViewController: UIViewController {
+    private var searchTextField: UISearchBar!
+    private var tableView: UITableView!
+    private let cities = ["Łódź", "Warszawa", "Kraków", "Wrocław", "Gdańsk", "Gdynia", "Bydgoszcz", "Sopot", "Kielce"]
+    private var dataToDispaly: [String]!
+
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        view.backgroundColor = .systemCyan
+
+        dataToDispaly = cities
+
+        configureTextField()
+        configureTable()
+    }
+}
+
+extension SearchTabViewController {
+    private func configureTextField() {
+        searchTextField = UISearchBar()
+        searchTextField.barTintColor = .systemCyan
+        searchTextField.layer.cornerRadius = 15.0
+        searchTextField.placeholder = "Search City"
+        searchTextField.searchBarStyle = .minimal
+        searchTextField.delegate = self
+
+        view.addSubview(searchTextField)
+
+        searchTextField.snp.makeConstraints { make in
+            make.height.equalTo(60)
+            make.leading.trailing.equalToSuperview().inset(16)
+            make.top.equalTo(view.snp_topMargin)
+        }
+    }
+
+    private func configureTable() {
+        tableView = UITableView(frame: .zero, style: .insetGrouped)
+        tableView.backgroundColor = .systemCyan
+        tableView.dataSource = self
+        tableView.delegate = self
+
+        tableView.register(UITableViewCell.self, forCellReuseIdentifier: "cell")
+
+        view.addSubview(tableView)
+
+        tableView.snp.makeConstraints { make in
+            make.leading.right.equalToSuperview().inset(5)
+            make.top.equalTo(searchTextField.snp_bottomMargin).inset(-30)
+            make.bottom.equalTo(view.snp_bottomMargin).inset(10)
+        }
+    }
+}
+
+extension SearchTabViewController: UITableViewDataSource {
+    func tableView(_ tableView: UITableView, cellForRowAt indexPath: IndexPath) -> UITableViewCell {
+        let cell = tableView.dequeueReusableCell(withIdentifier: "cell", for: indexPath)
```

Comment:
you can use the same mechanism like in Raven to don't use custom strings as cell id

Reply:
👍